libxrdp/xrdp_sec.c

```diff
@@ -1603,7 +1605,8 @@ xrdp_sec_process_mcs_data_CS_CORE(struct xrdp_sec* self, struct stream* s)
     in_uint8s(s, 2); /* SASSequence */
     in_uint8s(s, 4); /* keyboardLayout */
     in_uint8s(s, 4); /* clientBuild */
-    in_uint8s(s, 32); /* clientName */
+    unicode_in(s, 30, 32, clientName, 15); /* clientName */
+    log_message(LOG_LEVEL_ALWAYS ,"client computer name: %s", clientName);
```

Comment:
Formatting: space goes after comma, not before. Why LOG_LEVEL_ALWAYS? It's for messages more important than LOG_LEVEL_ERROR (i.e. fatal errors). This condition looks like LOG_LEVEL_INFO to me - more important than debug information, but not something to worry about.